nautobot/dcim/tests/test_filters.py

```diff
@@ -1087,7 +1087,7 @@ def test_outer_unit(self):
         with self.subTest():
             self.assertEqual(Rack.objects.exclude(outer_unit="").count(), 3)
         with self.subTest():
-            params = {"outer_unit": RackDimensionUnitChoices.UNIT_MILLIMETER}
+            params = {"outer_unit": [RackDimensionUnitChoices.UNIT_MILLIMETER]}
```

Comment:
Should be able to change most of these affected tests to entries in `generic_filter_test` rather than separate test functions now that these are multi-value filters, though it may take a bit of modifying to the test data to make it diverse enough.

Reply:
It does not need to be done on this PR right?

Reply:
I won't block this PR on waiting for this change but it should be a pretty straightforward one to do. Fine with me if you want to open a new issue or add a comment on #3213 calling out these tests.